### PR TITLE
docs(README): simplify & sort roadmap points

### PR DESCRIPTION
**Pull request by zetok**
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/208)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
...
 
 ### Done
-- [X] Create Toxcore
 - [X] Create DHT
+- [X] Create Messenger
 - [X] Create Onion
+- [X] Create Toxcore
 - [X] Implement Crypto
-- [X] Create Messenger
+- [X] Make toxcore stateless
```
Comment (GrayHatter) [inline README.md]: This here implies all of Toxcore is already stateless, it's not.

File: README.md [hunk 2]
```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
...
+- [ ] Improve group chat implementation
+- [ ] Make ToxAV stateless
+- [ ] Multiple device support
 
 ### Done
-- [X] Create Toxcore
 - [X] Create DHT
+- [X] Create Messenger
 - [X] Create Onion
+- [X] Create Toxcore
```
Comment (GrayHatter) [inline README.md]: If you're adding these here, the order should be chronological. Or is there a better order that this into that I don't know?

File: README.md [hunk 3]
```diff
@@ -60,4 +58,3 @@ However, Tox does use [TCP relays](/docs/TCP_Network.txt) as a fallback if it en
 
 ### Connecting & Communicating
 Every peer is represented as a [byte string](https://en.wikipedia.org/wiki/String_(computer_science)) (the public key [Tox ID] of the peer). By using torrent-style DHT, peers can find the IP of other peers by using their Tox ID. Once the IP is obtained, peers can initiate a [secure](/docs/updates/Crypto.md) connection with each other. Once the connection is made, peers can exchange messages, send files, start video chats, etc. using encrypted communications.
-
```
Comment (GrayHatter) [inline README.md]: No blank line at eof?

File: README.md [hunk 4]
```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
 
 ### In Progress
```
Comment (GrayHatter) [inline README.md]: In this format, I don't think any/all are actually in progress, just pending.

File: README.md [hunk 5]
```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
...
-    - [ ] 100% unit testing
-    - [ ] Make ToxAV stateless
-    - [ ] Allow a single toxcore instance to handle multiple keypairs (or 'clients')
-    - [ ] Consistent naming scheme throughout toxcore
-    - [X] Make toxcore stateless
-- [ ] Messenger
-    - [ ] Improve group chat implementation
-    - [ ] Improve A/V implementation
-    - [ ] Multiple device support
+- [ ] 100% unit testing
```
Comment (GrayHatter) [inline README.md]: Is there an order here?

File: README.md (hunk 2 above)
Comment (zetok) [inline README.md]: `sort`

File: README.md (hunk 4 above)
Comment (zetok) [inline README.md]: Should I change to `Tasks` ?

File: README.md (hunk 3 above)
Comment (zetok) [inline README.md]: Why would it be needed?

File: README.md (hunk 1 above)
Comment (zetok) [inline README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (SkyzohKey) [inline README.md]: @zetok Removing `EOF` seems stupid to me, that's a global convention, even in Markdown files and should continue to be.

File: README.md [hunk 6]
```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
```
Comment (SkyzohKey) [inline README.md]: If this has to be merged, you'd be better removing these lines, as they are inverse to your pull request.

File: README.md (hunk 3 above)
Comment (iphydf) [inline README.md]: `EOF` is not a thing. This change is replacing `\n\n` at the end of the file with `\n`, which is a good change.

File: README.md (hunk 4 above)
Comment (GrayHatter) [inline README.md]: `tentative roadmap` maybe? Or something similar to that. It's not really a task, but a group of tasks

File: README.md (hunk 5 above)
Comment (GrayHatter) [inline README.md]: alphanumeric order is stupid for a roadmap. Choose a different order

File: README.md (hunk 2 above)
Comment (GrayHatter) [inline README.md]: alphanumeric is not better than chronological

File: README.md (hunk 5 above)
Comment (zetok) [inline README.md]: It's not a roadmap though, given that tasks listed in previous disorder weren't being worked on one after another..?

Well, we can always do `sort -R` to bring back disorder.

File: README.md (hunk 2 above)
Comment (zetok) [inline README.md]: It can be if you want to automate sorting.

File: README.md (hunk 6 above)
Comment (zetok) [inline README.md]: Thanks, fixed.

**Comment by GrayHatter** [comment]
I dismissed my review, I disagree with the rewrite for the reasons I commented, but @zetok doesn't agree with my requests and has declined to make the requested changed.

Because this documentation only, with no LOC changed, I'm non-blocking at this point, but still not approving this pull.

**Comment by iphydf** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[README.md, line 15 at r1](https://reviewable.io:443/reviews/toktok/c-toxcore/208#-KVQWHDvF2sc39pQIgN2:-KVQWHDvF2sc39pQIgN3:b-x9w39w) ([raw file](https://github.com/toktok/c-toxcore/blob/9e48ab3cbaa1b1e7c76813bfba06a433f59b21cf/README.md#L15)):*
> ```Markdown
> we're going, and where we've been.
> 
> Sorted alphanumericaly, the following is intended to function as a discussion
> ```

alphanumerically

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/208)*

**Comment by iphydf** [comment]
@nurupo this is yours now.
@zetok fix typo.

**Comment by zetok** [comment]
um, I thought that change might be beneficial, but given the opposition I'm no longer sure, so I'm closing this.

**Pull request closed without merging**